publications/LCS1.md

```diff
@@ -0,0 +1,36 @@
+# LCS 1 - Definition of the Lightning Creations Specification Project
+
+Publication Date: 09-22-2019
+
+Copyright © 2019 Connor Horman and Lightning Creations. This document may be freely copied and distributed but changing is not allowed, except as explicitly licensed.
+
+## §1 Definition of the Project
+
+The Lightning Creations Specification Project is a project which seeks to publish various specifications pertaining to Lightning Creations Projects. The Specifications are published by Lightning Creations and the LCS Review Committee on behalf of the author of that specification. These specifications will be assigned some number, and can be referred to as LCS followed by the number. For example, the 1st LCS can be referred to as LCS 1. Additionally, LCS Drafts may be done freely, and can be referred to as LCSD followed by the name of person who is writing the Draft, followed by the number that person chose to assign to it. This will be reviewed for publication according to a process layed out in a future LCS. This process is herein referred to as the "LCS Review Process", and will be carried out by nominated members of Lightning Creations, herein referred to as the "LCS Review Committee". The publication of an LCS is not a statement of endorsement of that LCS by either Lightning Creations or the LCS Review Committee. Neither Lightning Creations or the LCS Review Committee provides any warranties of any kind about Published LCS Documents, and may not be held liable for any damages caused, directly or indirectly, through the use of an LCS Document. 
+
+
+## §2 Structure of an LCS Document
+
+An LCS Document or LCS is formatted as a series of sections, where the _n_th section is indicated by a Heading labeled as §_n _followed by the given name of the section. Additionally, these sections may be divided into a series of subsections, subsections may be further subdivided, etc. There is no limit to the depth or number of these subdivisions. A subdivision, m levels deep, is labeled as §_n<sub>1</sub>_._n<sub>2</sub>_.(...)._n<sub>m</sub>_. Subdivisions of a section should be related to the level directly enclosing it. Further, sections of an LCS Document should be related to the Specification defined in it.

```

Comment:
Severely check your formatting on this. Several parts of it are not rendering properly, especially "_n_th" (etc).
"sections of an LCS Document should be related to the Specification defined in it" - Ambiguous use of "it". What are you referring to?

Reply:
Antecedents are complicated :man_shrugging: